api/app/GraphQL/Scalars/PhoneNumber.php

```diff
@@ -23,6 +23,6 @@ class PhoneNumber extends Regex
 
     public static function regex(): string
     {
-        return '/^\+[1-9]\d{1,14}$/';
+        return '/./';
```

Comment:
Could you update the description of this scalar?

Reply:
I wonder if its even worth having this scalar anymore if it's not really doing anything?

Reply:
I like the idea of keeping the scalar - even if its not validating anything, it makes it clear how to interpret the field.